src/data-access/MwApiLangCodeRetriever.ts

```diff
@@ -5,17 +5,22 @@ import { processWbGetClaimsResponse, WbGetClaimsResponse } from './apiBasedLangC
 export default class MwApiLangCodeRetriever implements LangCodeRetriever {
 
 	private readonly api: MwApi;
-	private readonly languageCodeProperty: string;
+	private readonly languageCodeProperty: string | null;
 
-	public constructor( api: MwApi, languageCodeProperty: string ) {
-		if ( typeof languageCodeProperty !== 'string' ) {
-			throw new Error( `Expected languageCodeProperty to be propertyId but received ${languageCodeProperty} instead!` );
+	public constructor( api: MwApi, languageCodeProperty: string | null ) {
+		if ( !languageCodeProperty ) {
+			// eslint-disable-next-line no-console
+			console.warn( 'LexemeLanguageCodePropertyId seems not configured, language code inference will not work.' );
 		}
 		this.api = api;
 		this.languageCodeProperty = languageCodeProperty;
 	}
 
 	public async getLanguageCodeFromItem( itemId: string ): Promise<string | null | false> {
+		if ( !this.languageCodeProperty ) {
+			return null;
```

Comment:
Should we update the documentation in `LangCodeRetriever` for this case? I think “Returns `null` if _there is_ no associated language code” (instead of “…if _the item has_ no…”) could work. (In that case we could change the end to “including missing or novalue statements”, or we could just leave that out entirely.)

Reply:
Done.

Reply:
ah, right, IIRC that line was at 99 characters or so :D